deployment/grid/terraform/compute_plane/auth.tf

```diff
@@ -37,34 +59,82 @@ resource "aws_cognito_user_pool_client" "client" {
     "ALLOW_REFRESH_TOKEN_AUTH"
   ]
 
+  depends_on = [data.kubernetes_ingress_v1.grafana_ingress]
```

Comment:
Is this dependancy  required because we used that data object directly at line 49 ?

Reply:
Nice catch! This was probably left here after testing the different variations of ensuring the explicit and implicit dependency handling and a misshap on cleanup. I will test if it works without this and remove it accordingly.